kolibri/core/content/api.py

```diff
@@ -555,12 +555,12 @@ def consolidate(self, items, queryset):
 
 
 class OptionalPagination(ValuesViewsetCursorPagination):
-    ordering = "id"
+    ordering = "lft"
```

Comment:
```suggestion
    ordering = ("lft", "id")
```
This seems to have minimal impact on performance and will give a unique ordering every time.